06_puppy/willshen8/storer_test.go

```diff
@@ -0,0 +1,86 @@
+package main
+
+import (
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/suite"
+)
+
+type StorerSuite struct {
+	suite.Suite
+	storer Storer
+	Puppy1 Puppy
+	Puppy2 Puppy
+	Puppy3 Puppy
+	Puppy4 Puppy
+	Puppy5 Puppy
+	Puppy6 Puppy
+	Puppy7 Puppy
+}
+
+func (s *StorerSuite) SetupTest() {
```

Comment:
Good to see the use of SetupTest() to define a test fixture that is consistent across all tests and not accidentally carry across changes from one test to another.
:+1: